### DISCUSSION TITLE
Question about ORP Drift

### DISCUSSION THREAD

**Discussion by subcritical71** [tagyoureit/nodejs-poolController#766] [General]
Hopefully I can gain some knowledge from the group to what I am observing on my pool chemistry. Last night I finally got my industrial AIO sensor plumbed, configured and commissioned. I only have a SWG at the moment but an acid tank is definitely in my future. Everything seems to be working well. The ORP was around 450 and rising after the first started up after calibration and pH was 7.8.  I don't think this is related but I did add 32oz of acid since my pH was so high. In the future I will not put so much in at one time. I don't normally run the pool pump except overnight for ~3.5 hours. And then if I add chemicals or there are swimmers I'll turn the pumps on. 

What I am observing and I don't know if it is normal or not is when the pumps are off my ORP drops. Below is a graph of what I'm seeing. The rapid drops are from when I turn the pumps off. The ORP went up slowly overnight to 866 before it started to slowly decrease to 855.  The rapid increases are from when I first turn a pump on. 

I do believe the ORP can be this high as my last water test over a week ago was at 6.6. I cleaned the SWG afterwords which seems to put it into turbo mode (it doesn't, but I assume since it does regain a lot of its effectiveness and I forget to turn it down my Ch goes high for awhile until I get it under control again - one of the reasons for installing a REM Chem!). 

Is this normal behavior? If not, any ideas on what may cause this? 

<img width="1285" alt="Screen Shot 2023-05-26 at 7 32 14 AM" src="https://github.com/tagyoureit/nodejs-poolController/assets/12250429/15f7fbe0-8c85-4787-a980-5b672275ef05">

**Comment by tagyoureit** [comment]
I would just watch this over a few days.  This could be perfectly normal.  ORP is the measure of the reduction potential of the chlorine.  When you add chemicals, this changes.  When the temperature changes, this changes.  When you have the pumps off, chlorine is being used and this changes.  Swimmers in the pool will change this.  A change in ~10 orp is insignificant, btw.  

> I don't normally run the pool pump except overnight for ~3.5 hours.

This isn't very long.  What size is your pool and what speed do you run your pumps?  21.5 hours of chlorine sitting at your ORP probe and not moving could certainly make a change this big in your reading.

**Comment by rstrouse** [comment]
Yeah what @tagyoureit said.

I too have peaks and valleys with ORP and I do believe it to be normal.  The measurement here is the oxidation reduction potential.  This is actually giving you a strong idea of how well the water can break down organics.  This varies wildly depending on conditions.  If you have cya (and if you live in a climate with tons of son you should have at least 30) then this potential is constantly getting bound and released.  

The effective bits and what you are measuring when you do a chlorine test are two compounds these are hypochlorous acid and hypochlorite.  Liquid chlorine exists primarily as a sodium hypochlorite.  You can actually watch this cycle occur if you add a gallon or two of chlorine to your pool.  The first thing it will do is shoot your pH up as it breaks down into hypochlorite ions then it will rapidly drop as it transitions to hypochlorous acid.  While this acid is relatively weak the hypochlorous state is the most reactive and has the greatest ability to sanitize (react with organic compounds).  The chlorine test does not differentiate the compounds or measure other oxidizing components of the pool.

The other thing is that chlorine is constantly reacting in the water.  It almost always exists in the pool as a transition from one compound to another.  Since, each compound has greater or lesser potential than others, it's potential changes wildly.  I can see it in the ORP when bathers get into the pool and I can see it when the sun comes out strong.  It takes minutes to break down chlorine with UV at which point it off-gases from the pool.  

I can't remember precisely what that compound is but I think it gets bound up and exits as chlorine dioxide which then raises the carbonate alkalinity in the pool as there is less CO2 to create carbonic acid.  The key is to watch the cycle and you will easily be able to see your pool's current health by that picture.  Nearly every peak and valley will have a reason and an instantaneous ORP reading doesn't mean much without thinking about the 4th dimension.  What time is it and what happened in past several hours.

**Reply by subcritical71** [reply]
Thanks both. I'll increase my turnover time as the pool is now starting to be used again, it is too low at the moment (pool is 21,000 gal.)

I guess there may be a request then. Since my ORP bottoms out (this morning I turned the pumps off and ORP is 336mV and no sign of leveling off after 5.5hrs), when I go to turn on my pool pump the REM Chem immediately turns on the SWG at 100% for 15 minutes. It would be nice to have a buffer similar to the liquid chlorine injection scenario so that I don't get that dose of Chlorine when it will stabilize at a much higher ORP once the pumps are running for ~10min.

**Comment by subcritical71** [comment]
I should have also mentioned that there was no liquid (storage solution) in the cap when I removed it. I thought I had seen some moisture drops when I removed it, but when I had completed the calibration I checked it again and it was bone dry. That is one of the reasons I'm a bit concerned.

**Reply by rstrouse** [reply]
Typically that is not a good thing.  I would contact Atlas.  If the glass dries out it will need to be reconditioned.  If you said it was moist maybe but bone dry is not good.
